src/flamegraph/merge.rs

```diff
@@ -176,11 +180,15 @@ where
         );
     }
 
+    if stripped_fractional_samples {
+        warn!("The fractional parts of sample counts have been stripped. If you need to retain the extra precision you can scale up the sample data and use the --factor option to scale it back down.");
```

Comment:
I think it's better for this warning to be printed the first time we see it rather than at the very end. Otherwise I might not realize until the program has completed that I need to re-run.